### ISSUE TITLE
info->insn_type not set on disassembled instructions

### ISSUE THREAD

**Issue by bluewww** [riscv/riscv-binutils-gdb#153]
`riscv_disassemble_insn` in `riscv-dis.c` sets info->insn_info_valid = 1 indicating that info->insn_type is useful. It is expected that insn_type reveals some information about the disassembled instruction. Currently it is always set to dis_nonbranch. It looks like that this feature is just not implemented yet.

**Comment by jim-wilson** [comment]
Yes, it hasn't been implemented yet, though it looks like most binutils disassemblers do not support this feature, so this looks like a low priority item to fix.

**Comment by bluewww** [comment]
I think as long as this functionality is not implemented ```info->insn_info_valid``` should not be set to a nonzero value. ```disassembler_info``` explains how it should be used.

**Comment by bluewww** [comment]
Thank you for implementing this feature!
